### DISCUSSION TITLE
Is it possible to install Mastodon on a custom domain?

### DISCUSSION THREAD

**Discussion by JulieAnndroid** [mastodon/mastodon#29109] [Q&A]
I am already a part of a Mastodon community. However, if I want my own Mastodon, can I install one on my own domain? If so, what type of hosting service would I need, VPS, for instance? Or a virtual machine? (A VM for me would be cheaper than investing in a physical server).👍And yes I searched for "custom domain" on here and didn't see what I was looking for.

**Comment by vmstan** [comment]
https://docs.joinmastodon.org/user/run-your-own/
